lib/doorkeeper/errors.rb

```diff
@@ -45,6 +49,14 @@ def self.name_for_response
       end
     end
 
+    class InvalidCodeChallengeMethod < BaseResponseError
+      def self.translate_options
+        {
+          challenge_methods: Doorkeeper.config.pkce_code_challenge_methods_supported.join(", ")
```

Comment:
Re: pluralisation in https://github.com/doorkeeper-gem/doorkeeper/pull/1747/files#r1825345954
```suggestion
          challenge_methods: Doorkeeper.config.pkce_code_challenge_methods_supported.join(", "),
          count: Doorkeeper.config.pkce_code_challenge_methods_supported.length
```

Reply:
Thanks, used this in ccc97732f3977e57d3a7c541689c30f418de6697